CardScan/Classes/ScanBaseViewController.swift

```diff
@@ -133,8 +133,12 @@ import Vision
     }
     
     @objc static public func cameraImage() -> UIImage? {
-        let bundleUrl = Bundle(for: ScanViewController.self).url(forResource: "CardScan", withExtension: "bundle")!
-        let bundle = Bundle(url: bundleUrl)!
+        guard let bundleUrl = Bundle(for: ScanViewController.self).url(forResource: "CardScan", withExtension: "bundle") else {
```

Comment:
similar to how you allowed resource and extensionName to be configured in `BundleURL`, would it be possible to have bundle resource loading go through that same path with a customiziable `bundleName`, `extensionName`?

Reply:
Yeah, agreed. I think we came up with something better and also provide the ability to set a bundle directly if you choose to do so. Check out the latest commit.